frontend/src/sharedComponents/Metadata/ContentComps.jsx

```diff
@@ -0,0 +1,297 @@
+        const comments = await res.json();
+        setCommentsCount(comments.length);
+      } catch (err) {
+        const { status } = err;
+        if (status === 404) {
+          dispatch(alertActions.error('Could not fetch comments count for dataset.'));
+        } else if (status === 401) {
+          dispatch(alertActions.info('Not authorized to see comments count for dataset.'));
+        } else {
+          dispatch(alertActions.error('Failed to fetch comments count. Please try again later.'));
+        }
+      }
+    };
+    getLikes();
+    getComments();
+  }, []);
+
+  const handleLike = async () => {
+    try {
+      const res = await fetch(`/api/Metadata/${uuid}/like`, {
+        method: 'PUT',
+        headers: {
+          Authorization: `bearer ${token}`,
+        },
+      });
+      const { status } = res;
+      if (status === 200) {
+        setLiked(!isLiked);
+        setLikes(isLiked ? likes - 1 : likes + 1);
+      } else if (status === 401) {
+        dispatch(alertActions.error('You must be signed in to like that dataset!'));
+      }
+    } catch (err) {
+      dispatch(alertActions.error('Something went wrong when adding/removing star.'));
+    }
+  };
+  return (
+    <ContentExpandedWrapper>
+      <MetaDataContent>
+        <MetaDataDescription>
+          <ReleaseStateLabel releaseState={releaseState} />
+          { showMunicipality ? (
+            <SmallLink to={`/municipalities/${municipalityName}`}>
+              <p>{municipalityName}</p>
+            </SmallLink>
+          ) : null }
+          { showCategory ? (
+            <SmallLink to={`/dataType/${metadataTypeName}`}>
+              <p>{metadataTypeName}</p>
+            </SmallLink>
+          ) : null }
+
+          <p>{description}</p>
+          <MetaDataLink to={`/dataset/${uuid}`}>See full entry</MetaDataLink>
+        </MetaDataDescription>
+        <MetadataURL url={url} formatName={formatName} />
+      </MetaDataContent>
+      <MetaDataRating>
+        <Favourite onClick={handleLike}>
+          {isLiked ? <StarFullStyled /> : <StarEmptyStyled />}
+          <p>{likes}</p>
+        </Favourite>
+        <CommentsLink to={`/dataset/${uuid}`}>
+          <CommentsIcon />
+          <p>{`${commentsCount} threads`}</p>
+        </CommentsLink>
+      </MetaDataRating>
+    </ContentExpandedWrapper>
+  );
+};
+ContentExpanded.propTypes = {
+  metadata: PropTypes.shape({
+    uuid: PropTypes.string,
+    formatName: PropTypes.string.isRequired,
+    url: PropTypes.string.isRequired,
+    description: PropTypes.string.isRequired,
+    releaseState: PropTypes.number.isRequired,
+    municipalityName: PropTypes.string.isRequired,
+    metadataTypeName: PropTypes.string.isRequired,
+    experiencePostGuid: PropTypes.string,
+  }).isRequired,
+  showCategory: PropTypes.bool,
+  showMunicipality: PropTypes.bool,
+};
+ContentExpanded.defaultProps = {
+  showCategory: false,
+  showMunicipality: false,
+};
+ContentCollapsed.propTypes = {
```

Comment:
There are two propTypes here now probably from the merging. Also experiencePostGuid doesn't exist, its replaced with
```
  experiencePosts: PropTypes.arrayOf(PropTypes.shape({
    experiencePostUuid: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
  })),
```

Reply:
Nvm, I can't read, but the experiencePostGuid still doesnt exist anymore